components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -578,15 +650,17 @@ export function RepositoryNotFound(p: { error: StartWorkspaceError }) {
         .toString();
 
     if (!userScopes.includes(missingScope)) {
-        return renderError(
-            "The repository may be private. Please authorize Gitpod to access private repositories.",
-            "Grant access",
-            authorizeURL,
+        return (
+            <RepositoryInputError
+                title="The repository may be private. Please authorize Gitpod to access private repositories."
+                linkText="Grant access"
+                linkHref={authorizeURL}
+            />
```

Comment:
**issue(non-blocking)**: Similar to the color change in the invalid configuration error, this is not a system error, correct? Previously this was equally confusing as the page looked like an error page, see https://github.com/gitpod-io/gitpod/issues/4696 and early designs from https://github.com/gitpod-io/gitpod/issues/4696#issuecomment-1167245638:

> | BEFORE | AFTER |
> |-|-|
> | <img width="1440" alt="Screenshot 2022-06-27 at 2 14 30 PM (2)" src="https://user-images.githubusercontent.com/120486/175930863-6231fe8c-48ec-4f8b-a518-0f3eb4e3344d.png"> | <img width="1440" alt="Screenshot 2022-06-27 at 2 34 02 PM (2)" src="https://user-images.githubusercontent.com/120486/175932408-bec2523d-b3ab-4732-9f04-e5c33f028663.png"> |